CONTRIBUTING.md

```diff
@@ -18,4 +19,4 @@ To stay on the list, projects should follow these quality standards:
 * Documented.
 * Tests.
 
-Thanks to all [contributors](../../graphs/contributors), you're awesome and wouldn't be possible without you!
+Thanks to all [contributors](https://github.com/DragonRidersUnite/awesome-dragonruby/graphs/contributors), you're awesome and wouldn't be possible without you!
```

Comment:
@kfischer-okarin sorry to waffle on this after merging it. I realized it broke the website links, like on https://www.dragonriders.community/awesome-dragonruby/

Reply:
No issues ;) I didn't think about it being published now!